bsip-0083.md

```diff
@@ -0,0 +1,74 @@
+
+**Update to Committee-Controlled BitAsset Issuer Fields**
+
+At the hardfork date, the committee-controlled BitAsset objects' issuer fields will be updated to specify the cn-vote account (ID 1.2.1148184). The existing price feeds will be cleared, and the cn-vote account will be specified as the feed producer. After the hardfork, the cn-vote account will be able to assign other feed producers at its discretion.
+
+**Disable the witness_fed_asset and committee_fed_asset Flags**
+
+At the hardfork date, the `witness_fed_asset` and `committee_fed_asset` flags will be unset from the BitAssets, and will be disabled from use in future assets.
+
+**Amend total_core_in_orders to Track Limit and Collateral Orders Separately**
+
+The `total_core_in_orders` field of the `account_statistics_object` will be deprecated, and assets in limit orders will be tallied in a new `total_core_in_limit_orders` field while collateral will be tracked in a new `total_core_in_collateral_orders` map, keyed on the collateralized asset ID. During vote weight calculation, the `total_core_in_limit_orders` will add directly to the account’s voting weight; additionally, the `total_core_in_collateral_orders` which exceeds the feed price value of the collateralized debt will be added to the account’s voting weight as well.
+
+**Enable Remittance of Worker Proposals in Core Asset**
+
+Escrow agents administering existing worker proposals presently denominated in BitAssets shall be authorized to remit worker payments in BitShares core asset, or in a BitAsset selected at the discretion of the workers and escrow agents.
+
+# Discussion
+### Voting Power
+The voting power of debt holders will be reduced to only the portion of collateral that exceeds full collateral valuation of the debt assets that are borrowed from the blockchain. It is arguable who “owns” BTS that are locked up as collateral in BitAsset smart contracts. The authors of this BSIP hold the view that debt holders have received BitAssets in return for their collateral (which they may have sold on the market for more BTS), and thus have given up rights to the collateral itself, at least to the portion of collateral that represents the value of the BitAssets they have received. It is only logical that by exchanging the collateral for BitAssets they have given up the rights associated with the collateral, including its voting power.
+### Worker Proposals
+Worker proposals were originally denominated in BitAssets with the intent to add liquidity in their respective markets. With BitAssets now being placed under user control, liquidity is no longer the concern of the platform. A further implication of placing BitAssets under custodian control is that workers being paid in BitAssets would be exposed to the risk of devaluation and/or de-pegging at the custodian's discretion. This BSIP aims to remedy this by allowing different payout methods deemed less risky.
```

Comment:
* Originally denominated in bitassets to offer workers a stable source of income, not just liquidity reasons.
* Payment in BTS is more risky than holding bitassets; bitassets have decreased in value less this year than BTS has.
* I don't think this is neccessary to include in this BSIP.

Reply:
The stability promise of BitAssets has been destroyed by BSIP-76. bitCNY is effectively pegged to 0.22 BTS/bitCNY now, with the same volatility as BTS.

The goal of this BSIP is to decouple platform governance (including worker proposals) from BitAsset governance. Demanding payout in BitAssets for platform workers means they remain coupled.

Reply:
When the MSSR=110%, the bitasset still have the same volatility as BTS, but i didn't find you say that, why?

Reply:
That is not true. Only the spread was higher.

Reply:
Why is not true, when MSSR=110%, we have 20% volatility in bitasset.
When GS happen, this also destroyed the stability promise of BitAssets which has happend in bitusd.